### PR TITLE
fixed: the first letter of the fmt.ErrorF is not lowercase

### PR DESCRIPTION
**Pull request by orangegzx**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
> the first letter of the fmt.ErrorF is not lowercase

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
please sign-off the commit to pass the DCO.:)

**Comment by orangegzx** [comment]
> 请签署通过 DCO 的承诺。:)

**Comment by orangegzx** [comment]
> 请签署通过 DCO 的承诺。:)

**Comment by orangegzx** [comment]
> please sign-off the commit to pass the DCO.:)

Don't you need to work？ and you reply to the message so quickly

File: keadm/cmd/keadm/app/cmd/debug/get.go [hunk 1]
```diff
@@ -223,30 +223,30 @@ func (g *GetOptions) IsAllowedFormat(f string) bool {
...
-		return fmt.Errorf("Unrecognized resource type: %v. ", args[0])
+		return fmt.Errorf("unrecognized resource type: %v. ", args[0])
 	}
 	if len(g.DataPath) == 0 {
-		fmt.Printf("Not specified the EdgeCore database path, use the default path: %v. ", g.DataPath)
+		fmt.Printf("not specified the EdgeCore database path, use the default path: %v. ", g.DataPath)
 	}
 	if !isFileExist(g.DataPath) {
-		return fmt.Errorf("EdgeCore database file %v not exist. ", g.DataPath)
+		return fmt.Errorf("edgeCore database file %v not exist. ", g.DataPath)
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/debug/get.go:235]:
```suggestion
		return fmt.Errorf("edgecore database file %v not exist. ", g.DataPath)
```

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**